packages/react-native/lib/truvera-biometric-plugin.ts

```diff
@@ -101,6 +101,7 @@ async function issueMatchCredential(walletDID: string, enrollmentCredential: any
         issuer: truveraConfig.issuerDID,
         issuanceDate: getIssuanceDate(),
         expirationDate: expirationDate,
+        // schema: 'https://schema.dock.io/ForSurBiometricCheck-V4-1709846734949.json',
```

Comment:
?

Reply:
Forgot this while playing around with schemas. I'm working on a separate PR to resolve the biometric schemas linked to the ecosystem. Richard wants it to be configurable too

Reply:
I added a TODO comment, It will be handled in https://dock-team.atlassian.net/browse/DCKA-3245. I will push the PR for it later today